src/plugins/intel_cpu/src/nodes/rnn.h

```diff
@@ -161,6 +158,12 @@ class RNN : public Node {
     float inputScale    = 0.f;
     float inputShift    = 0.f;
     std::vector<float> weightsScales;
+
+    const uint64_t* m_gate_map;
+    // Need to reorder from the initial memory descs due to limited Reorders set.
+    MemoryPtr m_initial_weights[3] = { nullptr, nullptr, nullptr };
+    // Need to keep cache objects. Otherwise, they will be erased from the global cache.
+    std::unordered_set<MemoryPtr> m_weights_pull;
```

Comment:
shouldn't  internalBlobMemory keep these pointers?

Reply:
`internalBlobMemory` keeps the current weights only.